backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/bpartner/filter/BPartnerSimpleFuzzySearchFilterProvider.java

```diff
@@ -111,11 +123,14 @@ public String getSql(
 			return null;
 		}
 
-		// TODO: implement
-		// filter by: C_BPartner.Value, C_BPartner.Name, C_BPartner->C_BPartner_Location->C_Location.City etc (check the task description)
-		// use org.adempiere.db.DBConstants.FUNC_unaccent_string
+		return sqlOpts.getTableNameOrAlias() + "." + getSqlCriteria(sqlParamsOut, searchText);
 
-		throw new UnsupportedOperationException();
+	}
+
+	private String getSqlCriteria(final SqlParamsCollector sqlParamsOut, final String searchText)
```

Comment:
* consider using SqlAndParams
* don't forget using use org.adempiere.db.DBConstants.FUNC_unaccent_string when comparing strings. Basically the resulting SQL where clause shall look like 
```sql
unaccent_string(MyColumnName) ILIKE '%bla%bla%'
```

Reply:
Changed to using SqlAndParams and unaccent_string(?,1)